### PR TITLE
Update RDB Persistence Stage 1

### PR DESCRIPTION
**Pull request by vic-ma**

### CODE REVIEW

**Comment by rohitpaulk** [comment]
Hey @vic-ma - just realized we hadn't agreed on a process for requesting reviews. Is this ready for review? And going forward, can we do a quick comment that tags me for review whenever something is ready?

**Comment by vic-ma** [comment]
@rohitpaulk Yes, this is ready for review.

>And going forward, can we do a quick comment that tags me for review whenever something is ready?

Ok, got it.

File: course-definition.yml [hunk 1]
```diff
@@ -439,41 +439,67 @@ stages:
     name: "RDB file config"
     difficulty: easy
     description_md: |
-      Redis uses `.rdb` files for persistence. In this stage, you'll add support for reading the config values related to where RDB files are stored.
+      Welcome to the RDB Persistence Extension! In this extension, you'll add support for reading [RDB files](https://redis.io/docs/management/persistence/) (Redis Database files).
 
-      There are two config values that determine where RDB files are stored:
+      In this stage, you'll add support for two configuration parameters related to RDB persistence, as well as a command that returns the values of configuration parameters.
```
Comment (rohitpaulk) [inline course-definition.yml:444]:
```suggestion
      In this stage, you'll add support for two configuration parameters related to RDB persistence, as well as the [CONFIG GET](https://redis.io/docs/latest/commands/config-get/) command.
```

(Shorter + adds some "mystery" to the hook)

File: course-definition.yml [hunk 2]
```diff
@@ -439,41 +439,67 @@ stages:
...
 
-      - `*2\r\n` indicates that the array has two elements
-      - `$3\r\ndir\r\n` indicates that the first element is a bulk string with the value `dir`
-      - `$16\r\n/tmp/redis-files\r\n` indicates that the second element is a bulk string with the value `/tmp/redis-files`
+      ### Notes
 
-      **Note**: If your repository was created before 5th Oct 2023, it's possible that your `./spawn_redis_server.sh` script
-      might not be passing arguments on to your program. You'll need to edit `./spawn_redis_server.sh` to fix this, check
-      [this PR](https://github.com/codecrafters-io/build-your-own-redis/pull/89/files) for details.
+      1.  You don't need to do anything with the `dir` and `dbfilename` parameters—you only need to accept them.
```
Comment (rohitpaulk) [inline course-definition.yml:501]: A bit ambiguous - the user does need to "store" the parameters. How about: 

```suggestion
      1.  You don't need to read the RDB file in this stage, you only need to store `dir` and `dbfilename`. Reading from the file will be covered in later stages.
```

**Review by rohitpaulk — APPROVED** [review]

**Pull request merged**